src/bundle/Resources/views/themes/admin/ui/layout.html.twig

```diff
@@ -120,14 +120,53 @@
     <body class="{% block body_class %}{% endblock %}">
         <div id="react-udw" data-filter-subtree-udw-config="{{ ez_udw_config('single_container', {}) }}"></div>
 
-        {% block navigation %}
-            {% include '@ezdesign/ui/menu/top_navigation.html.twig' %}
+        {% block header_row %}
+            <div class="ibexa-main-header">
+                <div class="ibexa-main-header__brand-column">
+                    <a class="ibexa-main-header__brand" href="{{ url('ezplatform.dashboard') }}">
+                        <img class="ibexa-main-header__brand-image" src="/bundles/ezplatformadminui/img/ibexa-logo.svg" alt="Ibexa" />
```

Comment:
hmmm... translation? :D

Also, I'm not sure, don't we have some func for resources path?